### PR TITLE
added photos of products to resources and make css global file and make changes in css files

### PR DESCRIPTION
**Pull request by oripaso**
i wrote global css and connect this file to all css files, i made several changes in the css files.

### CODE REVIEW

**Review by oripaso — COMMENTED** [review]
add search function and fix the filter bar in the homepage

File: src/views/.DS_Store
Comment (yonatanHammer) [inline src/views/.DS_Store:1]: we don't need those file, make sure that you don't push them.

File: src/views/cssFiles/global.css
Comment (yonatanHammer) [inline src/views/cssFiles/global.css:1]: looks good, great job 👏

Comment (oripaso) [reply src/views/cssFiles/global.css:1]: tnx bro

**Pull request merged**